packages/gen-typescript-declarations/src/test/goldens/polymer-3/lib/legacy/templatizer-behavior.d.ts

```diff
@@ -114,37 +114,3 @@ interface Templatizer {
 }
 
 declare const Templatizer: object;
```

Comment:
This removes a lot of the type information that we previously had. Can we be smarter here and instead export the proper object type with the function definitions as they were previously below it?

Reply:
I think this is actually correct, and it's the same as what we have always done for behavior typings. For behaviors, we export two things with the same name: 1) an interface which represents how the behavior will augment a class, and 2) a plain object which is just the thing you pass to the behaviors array or the mixinBehaviors function. The behavior object itself shouldn't have methods on it, it's more like the constructor than the prototype.

Before this PR we were also incorrectly declaring all of the functions of the behavior as duplicated top-level functions, that's the only change.